Source/Core/Common/MemoryUtil.cpp

```diff
@@ -41,7 +41,9 @@ void* AllocateExecutableMemory(size_t size)
 #else
   int map_flags = MAP_ANON | MAP_PRIVATE;
 #if defined(_M_ARM_64) && defined(__APPLE__)
-  map_flags |= MAP_JIT;
+  // This check is in place to prepare for x86_64 MAP_JIT support.
```

Comment:
I wonder if it's worth noting here (for anyone who'd circle back) that `MAP_JIT`, while it did exist prior to `10.14`, had restrictions on the number of JIT blocks you could have, and that's why it needs to be blocked off? Old Apple docs can be a bit of a pain to dig up and I could see this being easy to think "it should be fine" since the flag existed before...

Reply:
Sure, I can add a little bit more info there.